<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupBrandingPage</class>
 <widget class="QWidget" name="SetupBrandingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="brandingHintLabel">
     <property name="text">
      <string>Название и логотип отображаются в заголовках окон и на печатных чеках. Адрес и реквизиты можно указать для чеков.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing"><number>8</number></property>
     <property name="horizontalSpacing"><number>12</number></property>
     <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
     <property name="labelAlignment"><set>Qt::AlignRight|Qt::AlignVCenter</set></property>
     <item row="0" column="0">
      <widget class="QLabel" name="appNameLabel">
       <property name="text">
        <string>Название приложения:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="appNameEdit">
       <property name="placeholderText">
        <string>easyPOS</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="logoLabel">
       <property name="text">
        <string>Логотип (файл):</string>
       </property>
       <property name="toolTip">
        <string>Путь к изображению (PNG, JPG). Необязательно.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="logoLayout">
       <item>
        <widget class="QLineEdit" name="logoPathEdit">
         <property name="placeholderText">
          <string>Путь к файлу логотипа</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseLogoButton">
         <property name="text">
          <string>Обзор…</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Адрес:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="addressEdit">
       <property name="placeholderText">
        <string>Адрес организации (на чеках)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="legalLabel">
       <property name="text">
        <string>Юридическая информация:</string>
       </property>
       <property name="toolTip">
        <string>ИНН, ОГРН и т.д. — для печатных чеков</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="legalInfoEdit">
       <property name="maximumHeight">
        <number>80</number>
       </property>
       <property name="placeholderText">
        <string>ИНН, ОГРН и др.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
